src/Hadolint/Rules.hs

```diff
@@ -311,33 +311,23 @@ useAdd = instructionRule code severity message check
     code = "DL3010"
     severity = InfoC
     message = "Use ADD for extracting archives into an image"
-    check (Copy src dst) = and [not (format `isSuffixOf` src) | format <- archive_formats]
-    check _ = True
-    archive_formats = [".tar", ".gz", ".bz2", "xz"]
-
-exposeMissingArgs = instructionRule code severity message check
-  where
-    code = "DL3021"
-    severity = ErrorC
-    message = "EXPOSE requires at least one argument"
-    check (Expose (Ports ports)) = not (null ports)
-    check (Expose (PortStr "")) = False
-    check _ = True
-
-copyMissingArgs = instructionRule code severity message check
```

Comment:
This also became a parse error

Reply:
Can you please also remove it from `README.md`?

Reply:
It was not in the readme!

Reply:
Sorry, my bad. I was expecting that all checks are in README.